### ISSUE TITLE
Folder-level comparison

### ISSUE THREAD

**Issue by niccolomineo** [atom-compare-files/atom-compare-files#54]
It would be great to have folder-level recursive comparison, to filter out identical files and show the differing ones only. It would be the first Atom diff package to sport something like this.

**Comment by ParitoshBh** [comment]
Could you throw some more light on its working, an example maybe?

**Comment by niccolomineo** [comment]
I have IntelliJ Idea's diff viewer in mind:
https://www.jetbrains.com/help/idea/comparing-folders.html

It's the most praised non standalone diff viewer on the web.

**Comment by nobodo** [comment]
Yes, I'd like to see this too.

Something like combination of atom-commander and split-diff.

**Comment by pingram3541** [comment]
Yes this is desperately needed when comparing project folders to know exactly what was changed between versions without having to check every single file.  One of the few areas Sublime is still more equipped.
